service/api/User.yaml

```
@@ -40,6 +40,7 @@ paths:
           $ref: '#/components/responses/403forbidden'
         '5XX':
           $ref: '#/components/responses/5XXserverError'
+
   /users/email/{email}:
```

parameter named `email` in path and `userEmail` in parameter definition
```suggestion
  /users/email/{userEmail}:
```